.github/ISSUE_TEMPLATE/Problem_with_resource.md

```diff
@@ -0,0 +1,73 @@
+---
+name: Problem with a resource
+about: If you have a problem, bug, or enhancement with a resource in this resource module.
+---
+<!--
+    Your feedback and support is greatly appreciated, thanks for contributing!
+
+    ISSUE TITLE:
+    Please prefix the issue title with the resource name, e.g.
+    'ResourceName: Short description of my issue'
+
+    ISSUE DESCRIPTION (this template):
+    Please provide information regarding your issue under each header below.
+    Write N/A under any headers that do not apply to your issue, or if the
+    information is not available.
+
+    NOTE! Sensitive information should be obfuscated.
+
+    PLEASE KEEP THE HEADERS.
+
+    You may remove this comment block, and the other comment blocks,
+    but please keep the headers.
+-->
+#### Details of the scenario you tried and the problem that is occurring
+
+#### Verbose logs showing the problem
+
+#### Suggested solution to the issue
+
+#### The DSC configuration that is used to reproduce the issue (as detailed as possible)
+```powershell
+# insert configuration here
+```
+
+#### SQL Server edition and version the target node is running
+<!--
+    To help with this information, please run this command in SQL:
+    select @@version
+-->
+
+#### SQL Server PowerShell modules present on the target node
+<!--
+    To help with this information, please run this command:
+    Get-Module -Name '*sql*' -ListAvailable | ? Name -ne 'SqlServerDsc' | ft Name,Version,Path
+-->
```

Comment:
Change this to VSTeam instead of SQLServer

Reply:
Done